j2eeapis/external/deployment-api-1.2-rev-1-license.txt

```diff
@@ -1,11 +1,10 @@
-Name: JSR 88
-License: CDDL-1.0
-OSR: 136590
-Description: J2EE Deployment APIs 1.1
-Version: 88
-Origin: Oracle (http://jcp.org/aboutJava/communityprocess/mrel/jsr088/index.html)
+Name: Java EE Deployment API
+License: CDDL-1.1
```

Comment:
According to the embedded `META-INF/LICENSE.txt` this is CDDL-1.0

Reply:
yeah  I saw that, but the license link provided in the maven repository points to CDDL-1.1. 
So I had used CDDL-1.1. I will change it to CDDL-1.0